### ISSUE TITLE
Gui should display nested classes under their containing classes

### ISSUE THREAD

**Issue by CharliePoole** [TestCentric/testcentric-gui#198]
Currently, the Gui displays nested classes entirely separately from the classes that contain them. It would be more natural to display them as children of the enclosing class. When the enclosing class is selected, all nested classes will execute.

If the enclosing class is not itself a test fixture, then there is no further affect.

If the enclosing class is a test fixture, then this will also impact the order of test execution, whether the Gui is used or not. Nested fixtures will execute only after the setup for the enclosing fixture completes. They will not execute if that setup throws an exception. The enclosing class teardown will execute only after all the teardowns for the nested fixture complete. If the enclosing fixture contains test cases in addition to the nested fixture, the order of execution between those cases and the fixture is not defined.

From https://bugs.launchpad.net/nunit-3.0/+bug/782983

**Comment by rowo360** [comment]
I don't think I've ever used a nested class as a TestFixture before :blush:
So, I quickly add one - here's a screenshot:

<img src="https://github.com/user-attachments/assets/c45e3a0c-aed7-4ecb-b6c2-2fe9f5518525" width="250">

The name refers a little to the nesting. And looking into the test XML file doesn't show up any further indication for the nesting. So I believe that this task could be a challenge, especially when it comes to determining the order of test execution.

The idea is actually correct, but it is probably only relevant in very rare cases. And in these cases, the users were satisfied with the existing behavior. So, I think we don't need this feature for 2.0 release. And going one step further and even closing the issue is also OK.

**Comment by CharliePoole** [comment]
Exactly... that's how it has always looked. We did that by design back in V2 because building the tree of namespaces was already pretty complex and so we didn't create a hierarchy.

In the framework, such nesting could be used to provide a lot more control over setup and teardown than is currently given using SetUpFixtures, but without a framework implementation, just displaying them differently in the Gui really is of no benefit, so I'm going to close it.
